<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigCalibrationTreeViewDialog</class>
 <widget class="QDialog" name="ConfigCalibrationTreeViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure callibaration tree view</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Icons.qrc">
    <normaloff>:/Icons/Application.ico</normaloff>:/Icons/Application.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Window name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="WindowNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>external process</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ProcessComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ConfigFilter" name="Filter">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ValueGroupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Values</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="ViewValuesCheckBox">
          <property name="text">
           <string>view values</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ViewAddressCheckBox">
          <property name="text">
           <string>view address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ViewDataTypeCheckBox">
          <property name="text">
           <string>view data type</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="DecimalRadioButton">
            <property name="text">
             <string>decimal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="HexadecimalRadioButton">
            <property name="text">
             <string>hexadecimal</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigFilter</class>
   <extends>QGroupBox</extends>
   <header>ConfigFilter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigCalibrationTreeViewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigCalibrationTreeViewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
